content/en/md_conversion_test_files/common_components/alert.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Alert
+---
+
+## Expected .md output
+
+Each alert is enclosed in `{% alert %}` and `{% /alert %}` tags, with a `level` attribute.
```

Comment:
@onemec In cases like this, do you want me to also echo the attributes as plaintext content, like having "Info alert" as the bolded first line or something? Or does that get in the way? You can always add it yourself later in parsing as well, but happy to provide it up front if it's useful